swagger.yaml

```diff
@@ -148,7 +269,7 @@ paths:
         200:
           description: "Information for area returned"
           schema:
-            $ref: '#/definitions/GetAreaResponse'
+            $ref: '#/definitions/Area'
```

Comment:
I think this was right, returning a `GetAreaResponse`?

Reply:
There is only one area returned as part of this so this is correct.

Reply:
But we seem to return a `GetAreaResponse` containing that area instead of simply an `Area`? https://github.com/ONSdigital/dp-population-types-api/blob/a771635b150d9e8089c3a2653c758e5e1a18e996/contract/areas.go#L42-L44

Reply:
Sorry Rafa I missed the word "area" when I was checking the response.  I've now updated the file.